feedTheMonster.ts

```diff
@@ -292,6 +296,7 @@ class App {
   }
 
   private reinitializeSceneHandler(dataModal: DataModal): void {
+    this.titleTextElement.style.zIndex = "2";
```

Comment:
Why hard code here? Why not in css file?

Reply:
no need of this line of code can be handled with pure css only, avoid using zindex with js until you have to use a DOM element on multiple scenarios. @woodsjohn-codev